### PR TITLE
Yd/battleship-ZK

### PR DESCRIPTION
**Pull request by YauheniDraichykau**

### CODE REVIEW

File: frontend/apps/battleship-zk/.env.example [hunk 1]
```diff
@@ -0,0 +1,8 @@
+VITE_NODE_ADDRESS=
+VITE_CONTRACT_ADDRESS=
+VITE_GASLESS_BACKEND_ADDRESS=
```
Comment (vraja-nayaka) [inline frontend/apps/battleship-zk/.env.example:3]: Add VITE_ZK_PROOF_BACKEND_ADDRESS here please

File: frontend/apps/battleship-zk/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+COPY ./frontend/packages ./packages
+
+RUN apk update
+
+RUN apk add xsel
+
+ARG VITE_NODE_ADDRESS \
+    VITE_CONTRACT_ADDRESS \
+    VITE_SENTRY_DSN \
+    VITE_GASLESS_BACKEND_ADDRESS
```
Comment (vraja-nayaka) [inline frontend/apps/battleship-zk/Dockerfile:20]: And here VITE_ZK_PROOF_BACKEND_ADDRESS

File: frontend/apps/battleship-zk/.env.example (hunk 1 above)
Comment (YauheniDraichykau) [reply frontend/apps/battleship-zk/.env.example:3]: Added it in version with multiplayer

File: frontend/apps/battleship-zk/Dockerfile (hunk 2 above)
Comment (YauheniDraichykau) [reply frontend/apps/battleship-zk/Dockerfile:20]: Added it in version with multiplayer

**Comment by vraja-nayaka** [comment]
https://github.com/gear-foundation/dapps/pull/420

**Pull request closed without merging**